app/views/main/index.html.erb

```diff
@@ -1,13 +1,17 @@
+<%# Recommended keywords tag length: up to 255 characters, 20 words. %>
+<% set_meta_tags keywords: %w[writing author nanowrimo novel character fiction fantasy universe creative dnd roleplay larp game design] %>
+
 <div class="row">
   <div class="col s12">
     <div class="card">
       <div class="card-image">
         <%= image_tag 'card-headers/hero.jpg', width: '100%' %>
+        <% set_meta_tags image_src: image_url('card-headers/hero.jpg') %>
         <span class="card-title">Your digital notebook is here.</span>
       </div>
       <div class="card-content">
         <h4>
-          Notebook is a set of tools for writers, game designers, and roleplayers to create magnificent universes &ndash; and everything within them.
+          <%= description 'Notebook is a set of tools for writers, game designers, and roleplayers to create magnificent universes &ndash; and everything within them.' %>
```

Comment:
That's provided by the `meta-tags` gem. It sets the page's description, and then lets the string pass through.